packages/fluent/modules/widgets/core/FluentUIButton.jsx

```diff
@@ -1,5 +1,6 @@
-import React from "react";
+import React, { useState } from "react";
 import { IconButton, ActionButton, CommandBarButton, DefaultButton } from "@fluentui/react";
+import JSONEditorModal from './JSONEditorModal';
```

Comment:
You should not change `fluent` package for this task, only demo app should be affected